packages/client/examples/node.js/storeCar.js

```diff
@@ -0,0 +1,38 @@
+import fs from 'fs'
+import { pack } from 'ipfs-car/pack'
+import { NFTStorage } from '../../src/lib.js'
+import { Blob } from '../../src/platform.js'
+
+const endpoint = 'https://api.nft.storage' // the default
+const token = 'API_KEY' // your API key from https://nft.storage/manage
+
+async function main() {
+  const storage = new NFTStorage({ endpoint, token })
+
+  // locally chunk'n'hash the file to get the CID and pack the blocks in to a CAR
+  const { root, out } = await pack({
```

Comment:
We are complicating things here because of the test setup with Node, given that `ipfs-car` already offers a `packToBlob` function (for browser). Wondering if we should have also a browser examples folder?

Reply:
there is a browser examples folder! I can add a browser example once this PR is generally acceptable.

Reply:
@vasco-santos can has `packToBlob` for Node.js using `@web-std/blob`?

Reply:
Yes please for a browser example 😄

Reply:
> @vasco-santos can has packToBlob for Node.js using @web-std/blob?

@olizilla and me already talked about it. Having Node.js using @web-std/blob would mean we should be consistent and use it for the Browser as well. But then, we need to direct people to use @web-std/blob when they already know of Blob in the browser. What do you think @alanshaw ?

Reply:
hmm, why do you need to use `@web-std/blob` in the browser as well?

The nft.storage client only uses `@web-std/*` in Node.js:

browser: https://github.com/ipfs-shipyard/nft.storage/blob/main/packages/client/src/platform.web.js
node: https://github.com/ipfs-shipyard/nft.storage/blob/main/packages/client/src/platform.js

Reply:
Here it is: https://github.com/vasco-santos/ipfs-car/pull/25